Build fails because checksum for osmfilter.c changed

[Travis build log](https://travis-ci.org/awendt/familienradwege/builds/641992410)

```
shasum -c checksumfile
osmconvert.c: OK
osmfilter.c: FAILED
shasum: WARNING: 1 computed checksum did NOT match
Makefile:16: recipe for target 'integrity' failed
make[1]: *** [integrity] Error 1
make[1]: Leaving directory '/home/travis/build/awendt/familienradwege/tools'
Makefile:77: recipe for target 'tools/osmfilter' failed
make: *** [tools/osmfilter] Error 2
```

--- a/tools/checksumfile
+++ b/tools/checksumfile
@@ -1,2 +1,2 @@
 e67bf5c37fd07bbcfb8137a6d9b226f00b04e1c0  osmconvert.c
-327a434d8444d4a6044d9329e46126aa3151f057  osmfilter.c
+12e6a9c8cec37e8c0bef9c83338abd0ed9e539e9  osmfilter.c

